benchcab/utils/__init__.py

```diff
@@ -6,8 +6,13 @@
 import json
 import yaml
 import os
+import sys
```

Comment:
The `sys` module doesn't look like it is being used here. Can you run your changes through a linter?

Reply:
Apologies, I thought they were being triggered. Will test.